diff --git a/mobile/build.gradle.kts b/mobile/build.gradle.kts
index 2bfb09a..7d87650 100644
--- a/mobile/build.gradle.kts
+++ b/mobile/build.gradle.kts
@@ -17,8 +17,8 @@ android {
         applicationId = "com.shihab.diplay.cn"
         minSdk = 28
         targetSdk = 37
-        versionCode = 72
-        versionName = "0.2.12-cn.8"
+        versionCode = 74
+        versionName = "0.2.12-cn.8-ipv4.1"
 
     }
 
